tests/e2e_testdata.go

```diff
@@ -0,0 +1,514 @@
+			method:               "eth_getCode",
+			blockchain:           "3",
+			blockchainSubdomain:  "avax-mainnet",
+			origin:               "https://app.test2.io",
+			elapsedTime:          0.15785,
+		},
+		{
+			applicationPublicKey: "12345fe0fc623aff8bba4ba1984e0c521c08874c9519cc6dcd518a15dd241f53",
+			nodePublicKey:        "test_node_pub_key_f64fe10e37173c0e7117490291c11e81d8851175e7ad4d",
+			method:               "eth_getTransactionCount",
+			blockchain:           "3",
+			blockchainSubdomain:  "avax-mainnet",
+			origin:               "https://app.test2.io",
+			elapsedTime:          0.054675,
+		},
+		{
+			applicationPublicKey: "12345d9470aef46d0ebd3cb3076f3a5a3228c650e374c69b3665b0e5b0017a69",
+			nodePublicKey:        "test_node_pub_key_498b2b95ba2db09a263314e5fe23ce94fbd555d23578c1",
+			method:               "eth_getStorageAt",
+			blockchain:           "3",
+			blockchainSubdomain:  "avax-mainnet",
+			origin:               "https://app.test2.io",
+			elapsedTime:          0.1093,
+		},
+		{
+			applicationPublicKey: "123455d57036c3bb5bd0de0319771cfdb3b2a28d4d64e33a3a23e6dcd6057f55",
+			nodePublicKey:        "test_node_pub_key_66b14f98b70d7e77572a6c72db611ebe28cf75d9cf542f",
+			method:               "eth_blockNumber",
+			blockchain:           "4",
+			blockchainSubdomain:  "bsc-mainnet",
+			origin:               "https://app.test3.io",
+			elapsedTime:          0.2205,
+		},
+		{
+			applicationPublicKey: "12345f9fe17a1f6aaca7fe9df6499e569ae2f4910708f636beb1efa20cebda4d",
+			nodePublicKey:        "test_node_pub_key_b5c0fd8910bb7a121686865407e1a3ba40dcfb700d5e87",
+			method:               "eth_getBalance",
+			blockchain:           "4",
+			blockchainSubdomain:  "bsc-mainnet",
+			origin:               "https://app.test3.io",
+			elapsedTime:          0.0932,
+		},
+		{
+			applicationPublicKey: "12345019c3f109073c77cd6d8bca9d1ff21b1ad5328ba04a5a610ba1bd72e1c5",
+			nodePublicKey:        "test_node_pub_key_124d8dcca024f05e01fb56c91800480926d62da44b0eee",
+			method:               "eth_blockNumber",
+			blockchain:           "4",
+			blockchainSubdomain:  "bsc-mainnet",
+			origin:               "https://app.test3.io",
+			elapsedTime:          0.1162,
+		},
+		{
+			applicationPublicKey: "1234546a9d0c7d69ac65ffcd508b068ea2651e5d4bd5f4760bd2643584b9ef6d",
+			nodePublicKey:        "test_node_pub_key_dbd1105f79c1c40c2fc8d8749ea5760700cff6a0504016",
+			method:               "eth_blockNumber",
+			blockchain:           "49",
+			blockchainSubdomain:  "fantom-mainnet",
+			origin:               "https://app.test1.io",
+			elapsedTime:          0.0814,
+		},
+	}
+)
+
+const (
+	// Test consts
+	phdBaseURL        = "http://localhost:8090"
+	apiKey            = "test_api_key_6789"
+	relayMeterBaseURL = "http://localhost:9898"
+	testUserID        = "12345678fgte0db3b6c63124"
+	mainBucket        = "mainnetRelay"
+	main1mBucket      = "mainnetRelayApp1m"
+
+	// Influx Tasks
```

Comment:
These strings MAY be more maintainable/readable if put into separate text files that are simply read here, to avoid embedding all of this into GO code (less certain about this though than with the test relays above)

Reply:
Again I am fine if you want to change it but my preference is everything is one file.

Reply:
nm, I moved them to a JSON file